e2e/libs/utility/globalVars.py

```diff
@@ -0,0 +1,38 @@
+import configparser
+import logging
+
+from kubernetes import client
+
+CONFIG_FILE_PATH = 'settings.ini'
+
+
+def initialize_variables():
+    logging.info("initiate environment variables")
+
+    global variables
```

Comment:
What difficulties we will encounter if there are no these global variables?

Reply:
I've created a wrapper class: `e2e/libs/utility/globalVars.py` around the config dict instead of much global variables.

Reply:
The same `global` variables are still here in the `globalVars.py`.

Can you help to look into how it works in our existing test code? It also uses k8s client api for sure and doesn't need to make it explicitly global.

Also the naming `globalVars.py` seems not right. Should we use snake case here?

And if this class is to deal with k8s client api, the name `globalVars` seems not suitable for it.

Reply:
Like https://github.com/longhorn/longhorn-tests/pull/1300#discussion_r1276056557, I've remove the `global` variables here. And I also rename `globalVar` as `global_config` which is only related to configuration.